plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/edit.tsx

```diff
@@ -41,10 +36,11 @@ function Pills( {
 				<li
 					key={ option.value }
 					className={ clsx(
-						'wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill',
+						'wp-element-button wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill',
 						{
 							'wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill--selected':
 								index === 0,
+							'is-style-outline': index !== 0,
```

Comment:
I'm unsure if we should be using this class. This is used by core as a variation of the button block, not the element. We should probably have a Woo specific class instead.